### PR TITLE
close #8 image recognition

### LINKED ISSUE #8: Image recognition
As a lazy festival goar,
So that I can find out about festivals just by taking a photo,
I want to be able to use the photo to find out the festival from it’s poster.

### PR DESCRIPTION
**Pull request by robota-x**

### CODE REVIEW

File: backend/models/ocr.js [hunk 1]
```diff
@@ -1,8 +1,124 @@
...
+      return searchForAllArtists(firstGroup)
+        .then(function(list) {
+          return list.map(function(val) {
+            return val.name;
+          });
+        });
+    });
+};
+
+parseImagetoBlob = function(url) {
```
Comment (eripheebs) [inline backend/models/ocr.js:26]: is blob the ocr terminology 😆

**Comment by eripheebs** [comment]
looks fabulous :shipit:

**Comment by jazzygasper** [comment]
:shipit:

File: backend/models/ocr.js [hunk 2]
```diff
@@ -1,8 +1,124 @@
...
+    });
+  });
+};
+
+parseLines = function(inputJSON) {
+  return JSON.parse(inputJSON)
+    .ParsedResults[0].TextOverlay.Lines
+    .map(function(entry){
+      var wordList = entry.Words.map(function(word) {
+        return word.WordText;
```
Comment (JojSh) [inline backend/models/ocr.js:97]: long function, can functionality be extracted/refactored at all?

Comment (robota-x) [inline backend/models/ocr.js:97]: yes

Comment (robota-x) [inline backend/models/ocr.js:97]: To elaborate further, many of the functions are hacked together to be able to test the rest of the product, so refactoring all this stuff is on the list right now

**Pull request merged**